deploy/ui/form.json

```diff
@@ -993,6 +1038,51 @@
                   "type": "text",
                   "description": "The name to use for this KIE Server."
                 },
+                {
+                  "label": "Enable Edge termination",
+                  "description": "If checked, the route will use edge termination (http).",
+                  "required": false,
+                  "jsonPath": "$.spec.objects.servers[*].terminationRoute.enableInsecureEdgeTermination",
+                  "type": "object",
+                  "fields": [
+                    {
+                      "label": "Port on Edge termination",
+                      "type": "dropDown",
+                      "required": false,
+                      "description": "Port to use with edge termination .",
+                      "jsonPath": "$.spec.objects.servers[*].terminationRoute.protocol",
```

Comment:
why expose it here if http seems to be the default value for edge setup?

Reply:
Because in the origianl PR was available https and the route can be created with https and edge, but edge doesn't work with https, can be removed now